### PR TITLE
Refactoring of character validation and Coverage Test Changes

### PR DESCRIPTION
**Pull request by donnachaholmes**
Moved the character validation class to be included in the uk_postcode_validation class
Removed the character_validation file as no longer needed
Added functionality to test class to allow successful coverage testing

### CODE REVIEW

**Comment by dodie91** [comment]
Name                       Stmts   Miss  Cover
----------------------------------------------
postcodes_test.py             91      0   100%
uk_postcode_validator.py      81      1    99%
----------------------------------------------
TOTAL                        172      1    99%

**Comment by dodie91** [comment]
### Coverage Report ###

| Name | Stmts | Miss | Cover|Missed|
| --- | --- | --- | --- | --- |
| postcodes_test.py | 91 | 0 | 100% | |
| uk_postcode_validator.py | 81 | 0 | 100% | |
| TOTAL | 172 | 0 | 100% | |

File: uk_postcode_validation/uk_postcode_validator.py [hunk 1]
```diff
@@ -41,3 +83,84 @@ def format(self, postcode):
...
+        length = len(postcode)
+        if length > 7 or length < 5:
+            return False
+        return True
+
+    def _validate_postcode_entries(self, postcode):
+        postcode = "".join(postcode.split())
+        outward_code = postcode[:-3]
+        inward_code = postcode[-3:]
+        if self._check_outward_code(outward_code) and \
```
Comment (alanholmes10) [inline uk_postcode_validation/uk_postcode_validator.py]: If you check the outward code first then it that is true check the inward, you will reduce the work done

**Comment by donnachaholmes** [comment]
### Coverage Report ###

| Name | Stmts | Miss | Cover|
| --- | --- | --- | --- |
| postcodes_test.py | 29 | 0 | 100% |
| uk_postcode_validator.py | 81 | 0 | 100% |
| TOTAL | 110 | 0 | 100% |

File: uk_postcode_validation/uk_postcode_validator.py (hunk 1 above)
Comment (dodie91) [reply uk_postcode_validation/uk_postcode_validator.py]: Nice, I think you can use just & and it does the same but I’ll test anyway, thanks!

**Pull request merged**